README.md

```diff
@@ -4,6 +4,10 @@
 
 This is an AngularJS 1.x directive which renders a slide wheel in canvas.
 
+## Demo
+
+You can view a demo of the slide wheel at <https://ayogohealth.github.io/ay-slide-wheel/>
```

Comment:
I don't think links in Markdown need to be wrapped in `<` and `>`. Just the URL itself should auto-link.